packages/synapse-react-client/src/lib/style/base/_core.scss

```diff
@@ -1320,10 +1320,10 @@ main,
 
 .SRC-tag {
   border: 1px solid SRC.$primary-action-color;
-  border-radius: SRC.$button-border-radius;
+  border-radius: 100px;
   padding: 5px 10px;
   color: SRC.$primary-action-color;
-  text-transform: SRC.$button-text-transform;
+  text-transform: uppercase;
 }
```

Comment:
Revert styling of tags, see "ANTIBODY": 
<img width="1242" alt="feature_FeaturedToolsList" src="https://user-images.githubusercontent.com/26949006/231906786-5dbd0138-afd2-4758-a21e-792bb08e3e14.png">